src/setup.py

```diff
@@ -36,10 +36,13 @@
     license='MIT',
     packages=find_packages(),
     install_requires=[
-        'ansible>=2.9',
+        'jupyterhub>=1.1.0',
         'jupyterhub-ltiauthenticator>=0.4.0',
+        'dockerspawner==0.11.1',
```

Comment:
`dockerspawner` already requires `docker` so I don't think we need to add the python `docker` client as a requirement.